### PR TITLE
DTSPO-8837 - Add codeowners

### PR DESCRIPTION
**Pull request by endakelly**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-8837

### Change description ###
Add codeowners

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+CODEOWNERS @hmcts/platform-operations
+
+# Restricting access to platops will we cleanup
+apps/ @hmcts/platform-operations
+clusters/ @hmcts/platform-operations
+
+# Production applications
+## environment rules
+k8s/environments/prod/ @hmcts/production-apps-approvals
```
Comment (louisehuyton) [inline CODEOWNERS:12]: This team needs write access to the repo.

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# https://help.github.com/en/articles/about-code-owners
+# Documentation: https://github.com/hmcts/sds-flux-config/blob/master/CODEOWNERS.md
+
+CODEOWNERS @hmcts/platform-operations
+
+# Restricting access to platops will we cleanup
```
Comment (timja) [inline CODEOWNERS:6]: what's this comment mean 'will we cleanup'

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# https://help.github.com/en/articles/about-code-owners
+# Documentation: https://github.com/hmcts/sds-flux-config/blob/master/CODEOWNERS.md
```
Comment (timja) [inline CODEOWNERS:2]: this file isn't in place, dunno if want to link to cnp-flux-config one or duplicate and change environment names?

File: CODEOWNERS (hunk 2 above)
Comment (endakelly) [reply CODEOWNERS:6]: Don't know tbh, used cnp-flux-config as a template and it has this

File: CODEOWNERS (hunk 3 above)
Comment (endakelly) [reply CODEOWNERS:2]: Yeah I can create one for sds-flux-config. Just copied from cnp-flux-config basically so it's another one of those that has been curated for that repo

File: CODEOWNERS (hunk 2 above)
Comment (timja) [reply CODEOWNERS:6]: prob just remove

File: CODEOWNERS (hunk 3 above)
Comment (timja) [reply CODEOWNERS:2]: sure

**Review by timja — COMMENTED** [review]
needs changes now that flux v1 / flux v2 is finished but looks pretty close

**Review by timja — APPROVED** [review]
let's start with this

**Pull request merged**